exercises/reverse-string/example.jl

```diff
@@ -0,0 +1,12 @@
+function myreverse(phrase::String)
+    chars = []
```

Comment:
Generally speaking it's best to not have arrays of type `Any`, which `[]` creates, for [performance reasons](https://docs.julialang.org/en/v1/manual/performance-tips/index.html#Avoid-containers-with-abstract-type-parameters-1).